### PR TITLE
added bsPhysics and baPhysics

### PR DESCRIPTION
**Pull request by mpark63**
B.A. Physics and B.S. Physics 

Screen Shot 2022-07-18 at 12 50 33 PM

### CODE REVIEW

File: lib/resources/majors.tsx [hunk 1]
```diff
@@ -1724,7 +2167,7 @@ const bsAMS: Major = {
           required_credits: 6,
           description: '<b>Computational Mathematics</b>',
           criteria:
-            'EN.553.481[C]^AND^(AS.110.445[C]^OR^EN.553.433[C]^OR^EN.553.467[C]^OR^EN.553.493[C])',
+            'EN.553.481[C]^OR^AS.110.445[C]^OR^EN.553.433[C]^OR^EN.553.467[C]^OR^EN.553.493[C]',
```
Comment (mpark63) [inline lib/resources/majors.tsx:2170]: Error I made in bsAMS

File: lib/resources/majors.tsx [hunk 2]
```diff
@@ -131,14 +131,460 @@ import { Major, Minor } from './commonTypes';
...
+        },
+        {
+          description: '<b>Neuroscience</b>',
+          required_credits: 12,
+          criteria: 'AS Neuroscience[D]',
+        },
+        {
+          description: '<b>Engineering</b>',
+          required_credits: 12,
+          criteria: '', // TODO: add WSE
```
Comment (mpark63) [inline lib/resources/majors.tsx:340]: criteria string for WSE? `200-600 level in  the School of Engineering (excluding courses listed as 500.xxx, 660.xxx, 551.xxx and 661.xxx)`

**Comment by kyang117** [comment]
For this TODO: "add WSE in criteria for bsPhysics electives (line 293 and 338)", apparently the "500.xxx, 660.xxx, 551.xxx and 661.xxx" are department codes.
You can reference them here: [https://e-catalogue.jhu.edu/arts-sciences/full-time-residential-programs/degree-programs/](url)
So, I'd prob expect the criteria to be something along the lines of listing the departments (the ones already listed and the 9 from WSE), minus the 4 exceptions.

**Comment by mpark63** [comment]
> 

wse string done

**Review by kyang117 — COMMENTED** [review]
don't forget to run prettier

**Review by kyang117 — COMMENTED** [review]
SORRY IM LATE

Small changes:
* on line 390 & 695 : "All Physics majors must [take] Calculus..."
* on line 441 & 746, are the sentences unfinished? Or are you purposely prompting the students to click on the distribution bars for more information when hovering?

Once you address these, you can consider me approved without actually having to wait for it this time ^^;; (and merge! Make sure to run Prettier again after too.) 
Great job! Very detailed work and lots of requirements haha

**Pull request merged**